src/postgraphile/http/createPostGraphileHttpRequestHandler.d.ts

```diff
@@ -1,101 +0,0 @@
-import { IncomingMessage, ServerResponse } from 'http'
```

Comment:
all of this content was trashed or moved into `interfaces.d.ts`, as referenced above